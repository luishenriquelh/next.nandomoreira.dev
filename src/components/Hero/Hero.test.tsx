import React from 'react';
import { render } from '@testing-library/react';
import { Hero } from '.';

describe('<Hero />', () => {
  test('Render component', () => {
    const componentTree = render(<Hero />);
    expect(componentTree).toMatchSnapshot();
  });

  test('Render component with children prop', () => {
    const componentTree = render(
      <Hero>
        <h1>Hello</h1>
      </Hero>,
    );
    expect(componentTree).toMatchSnapshot();
  });
});
