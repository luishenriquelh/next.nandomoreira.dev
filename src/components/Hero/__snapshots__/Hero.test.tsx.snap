// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Hero /> Render component 1`] = `
.c0 {
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  background-color: #272643;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  font-size: 2rem;
  height: 100%;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  min-height: 100vh;
  width: 100vw;
}

.c1 {
  color: #f6fafd;
  text-align: center;
}

<section
  className="c0"
>
  <div
    className="c1"
  >
    <h1>
      Hello Hero
    </h1>
  </div>
</section>
`;

exports[`<Hero /> Render component with prop text 1`] = `
.c0 {
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  background-color: #272643;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  font-size: 2rem;
  height: 100%;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  min-height: 100vh;
  width: 100vw;
}

.c1 {
  color: #f6fafd;
  text-align: center;
}

<section
  className="c0"
>
  <div
    className="c1"
  >
    Hello
  </div>
</section>
`;
